docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md

```diff
@@ -3,4 +3,35 @@ name: vercel
 routeTemplate: ./data/component-templates/article.yml
 title: Deploying to Vercel
 ---
-# Walkthrough: Deploying to Vercel
\ No newline at end of file
+# Walkthrough: Deploying to Vercel
+
+
+Vercel is an all-in-one platform with Global CDN supporting static & Jamstack deployment. Vercel has first-class support for Next.js.
+
+Using Vercel, you can easily deploy the sample app to production. 
+
+We assume you have set up the Next.js sample application to connect to a Sitecore instance for the following steps.
+
+To deploy the Next.js sample app to production: 
+1. Execute the steps provided in the Next.js  guide for [Getting Started](https://nextjs.org/docs/deployment#getting-started) with deployment.
+2. In the root directory of the application, in the `.env` file, add or modify:
+	* `PUBLIC_URL` - your Vercel deployment URL.
+	* `SITECORE_API_KEY` - your Sitecore API key is needed to build the app.
+	* `SITECORE_API_HOST` - your Sitecore API hostname.
+	* `JSS_EDITING_SECRET` - your secret token. The `JSS_EDITING_SECRET` is optional for deployments but necessary if you want to use the Experience Editor with your Next.js Vercel deployment. Read about [connecting your Next.js application to the Experience Editor](/docs/nextjs/experience-editor/walkthrough/en.md).
+3. Push the changes to your Git provider.
+
+## `publish:end` webhook invocation
+
+Using the `publish:end` webhook, you can automatically trigger static site generation, cache clearing, and other automated activities with webhook support.
+
+To enable the `Publish:end` hook, you must: 
+
+1. [Create a deploy hook](https://vercel.com/docs/more/deploy-hooks#creating-a-deploy-hook) in Vercel.
+1. On your Sitecore instance, edit `/App_Config/Sitecore/JavaScriptServices/Sitecore.JavaScriptServices.AppServices.PublishingWebHook.config.example`:
```

Comment:
Can we change these steps working under the assumption that the developer will be creating a new configuration patch, and adding it to their VS solution, and deploying it from there? Renaming and editing files in the web root is not a real-world workflow for the platform.

Reply:
@nickwesselman Changed this step, based on adding config to solution